.golangci.yaml

```diff
@@ -0,0 +1,111 @@
+# See the detailed configuration at
```

Comment:
Eu sei que durante o vídeo eu falei pra colocar a configuração padrão efetiva, mas que tal a gente mudar de estratégia e deixar somente os valores que diferem do padrão? Altere uma das opções, talvez a lista de linters, e remova por favor o que for igual ao default.